slick.remotemodel.ts

```diff
@@ -7,12 +14,12 @@
   function RemoteModel() {
     // private
     var PAGESIZE = 50;
-    var data = {length: 0};
+    var data: { [key: number]: any, length: number } = {length: 0};
```

Comment:
I feel like `data` is _actually_ an array, but I wanted the changes to be minimal so I chose this weird type instead.